src/marqo/core/inference/inference_models/abstract_embedding_model.py

```diff
@@ -35,11 +34,6 @@ def load(self):
         self._load_necessary_components()
         self._check_loaded_components()
```

Comment:
Why do we need to separate these two method?

Reply:
I was expecting that _check_loaded_components() is called separately